### ISSUE TITLE
ora2pg incorrect syntax errors when compared to AWS SCT tool

### ISSUE THREAD

**Issue by MichaelDBA** [darold/ora2pg#1123]
The article by DB-BEST calls out a lot of errors with ORA2PG when compared to AWS SCT.
[](https://www.dbbest.com/blog/comparing-ora2pg-with-aws-sct/#user-defined-type-declared-in-a-routine).

There appear to be a lot of them with exact script comparisons in this article.  Can you address whether these are fixed now in ORA2PG?

**Comment by darold** [comment]
I will try to give a quick look at this this weekend.

**Comment by MichaelDBA** [comment]
Yeah, this article has a bunch of side by side comparisons of what ora2p generates vs sct.
